README.md

```diff
@@ -134,3 +134,15 @@ Shepherd is written in TypeScript, which requires compilation to JavaScript. Whe
 Shepherd currently has minimal test coverage, but we're aiming to improve that with each new PR. Tests are written with Jest and should be named in a `*.test.ts` alongside the file under test. To run the test suite, run `npm run test`.
 
 We use [TSLint](https://github.com/palantir/tslint) to ensure a consistent coding style and to help prevent certain classes of problems. Run `npm run lint` to run the linter, and `npm run fix-lint` to automatically fix applicable problems.
+
+### Semantic Releases
```

Comment:
Could you update (or remove) the release documentation here: https://github.com/NerdWalletOSS/shepherd/blob/master/docs/releasing.md

Reply:
Will do.